### PR TITLE
Support stylelint@11

### PR DESCRIPTION
**Pull request by perrin4869**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Personally, I'd just bump the peerDependencies version and skip the whole thing for testing 10.x too. Beats the point of `npm ci` and the lock file.

The Node.js version is a good addition, though.

/CC @olegskl

**Comment by perrin4869** [comment]
@XhmikosR yeah, I thought about testing just stylelint 11, but thought I'd trying to avoid a breaking change

**Comment by XhmikosR** [comment]
IMO this should be a major version bump without the custom stuff to test stylelint 10.x. `npm ci` should be used on CI.

**Comment by XhmikosR** [comment]
@olegskl friendly ping

**Comment by olegskl** [comment]
I agree with @XhmikosR. We should probably do a major release of gulp-stylelint for every major release of Stylelint.

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "node": ">=8.15.1"
   },
   "peerDependencies": {
-    "stylelint": "^10.0.0"
+    "stylelint": "^11.0.0 || ^10.0.0"
```
Comment (XhmikosR) [inline package.json]: switch this to `"stylelint": "^10.1.0 || ^11.0.0"`

File: package.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
     "gulp-rename": "^1.4.0",
     "gulp-sourcemaps": "^2.6.5",
     "sinon": "^7.3.1",
-    "stylelint": "^10.0.0",
+    "stylelint": "^11.0.0",
```
Comment (XhmikosR) [inline package.json]: Bump this to `11.1.1`

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,11 @@
 sudo: false
 language: node_js
 node_js:
-  - 11
+  - 12
   - 10
-  - 9
   - 8.15.1
+env:
```
Comment (XhmikosR) [inline .travis.yml]: Please revert the `env` and `before_install` changes, i.e. keep only the Node.js version changes (which could be split BTW).

**Comment by perrin4869** [comment]
Done!
I will be gone on vacations from tomorrow, please push any necessary changes :)

**Comment by XhmikosR** [comment]
@perrin4869 please do `npm i` locally, commit and squash your patches into one.

**Comment by XhmikosR** [comment]
@olegskl can you can merge this and #115 and release a new version (major) please?

**Comment by XhmikosR** [comment]
Actually I pushed a few more patches; they should be fairly trivial to review. Feel free to CC me if needed in any of my PRs.

**Pull request merged**